hledger/hledger.cabal

```diff
@@ -98,11 +98,6 @@ flag debug
   manual: True
   default: False
 
-flag terminfo
-  description: On POSIX systems, build with the terminfo lib for detecting terminal width
-  manual: False
-  default: True
-
```

Comment:
Removing a build flag/variant is always good! I had the impression terminfo could be a useful lib though, is there nothing in there we need ? I guess we have replaced it by other things.